src/bundle/Resources/public/ts/shared/BaseBadge.ts

```diff
@@ -0,0 +1,31 @@
+import Base from './Base';
+
+const MAX_BADGE_VALUE = 99;
```

Comment:
Just an idea - you can make this as configurable in component. Both set in constructor from example `data-ids-max-value` on container and maybe also method setMaxValue